src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/cas1/Cas1PremisesTransformer.kt

```diff
@@ -26,12 +26,12 @@ class Cas1PremisesTransformer(
     )
   }
 
-  fun toPremiseBasicSummary(entity: PremisesEntity): Cas1PremisesBasicSummary {
-    val apArea = entity.probationRegion.apArea!!
+  fun toPremiseBasicSummary(entity: ApprovedPremisesBasicSummary): Cas1PremisesBasicSummary {
     return Cas1PremisesBasicSummary(
       id = entity.id,
       name = entity.name,
-      apArea = NamedId(apArea.id, apArea.name),
+      apArea = NamedId(entity.apAreaId!!, entity.apAreaName),
```

Comment:
we are sure `entity.apAreaId` is never null I assume? Double bang (`!!`) always scares me :)